easybuild/easyconfigs/s/SunPy/SunPy-1.1.4-foss-2021b.eb

```diff
@@ -0,0 +1,79 @@
+easyblock = 'PythonBundle'
+
+name = 'SunPy'
+version = '1.1.4'
+
+homepage = "https://sunpy.org/"
+description = """The community-developed, free and open-source solar data analysis environment for Python."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('aiohttp', '3.8.1'),
+    ('astropy', '3.2.3'),
```

Comment:
SunPy v1.1.3 `SunPy-1.1.3-foss-2019b-Python-3.7.4.eb` depends on astropy 4.0.1. Why is this depending on a lower astropy 3.2.3? Could this be upgraded to astropy 4 as well and avoid the divergence in dependencies?